src/admin/index.js

```diff
@@ -66,6 +66,26 @@ module.exports = ({
     instrumentationEmitter.emit(DISCONNECT)
   }
 
+  const listTopics = async () => {
```

Comment:
Let me know if an optional regular expression argument makes sense here to filter and return just the topics matching that re. It can be handled outside the library as well.